.gitignore

```diff
@@ -35,6 +35,7 @@ node_modules
 /tools_docker/test_docker/tracim.env
 /tools_docker/test_docker/test_config.env
 tools_docker/test_docker/__pycache__
+tools_docker/Debian_Uwsgi/send_summary_mails.sh
```

Comment:
Is it here just for dev period ?

Reply:
No, it's supposed to stay.
The files generated from a .sample are usually git ignored to avoid inadvertently committing them.

Reply:
added `tools_docker/Debian_New_Uwsgi/send_summary_mails.sh`